app/assets/scripts/views/assets-detail.js

```diff
@@ -238,14 +239,18 @@ class AssetsDetail extends React.Component {
   }
 
   render () {
-    const { vpromm, language, roadProps } = this.props;
-
+    const { vpromm, language, roadProps, roadGeo } = this.props;
     const disId = get(roadProps, 'data.district.id', null);
     const disName = get(roadProps, 'data.district.name', null);
     const provId = get(roadProps, 'data.province.id', null);
     const provName = get(roadProps, 'data.province.name', null);
     const hasGeometry = this.hasGeometry();
 
+    let featCenter = [0, 0];
+    if (roadGeo.fetched) {
+      featCenter = center(roadGeo.data.features[0]).geometry.coordinates;
```

Comment:
I changed it to be the center of all features. I think it's better than the bbox because depending on the length of the road we'll end up with different zoom levels. This way it gets centered to a point and the zoom level stays constant.
Let me know if this is ok